ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestRunnable.java

```diff
@@ -383,6 +384,21 @@ void startAgentsOn(PerfTest perfTest, GrinderProperties grinderProperties, Singl
 				+ " agents are ready.");
 	}
 
+	private boolean isValidStopReason(StopReason stopReason) {
+		PropertiesWrapper properties = config.getControllerProperties();
+		if (stopReason == StopReason.TOO_LOW_TPS &&
+			properties.getPropertyBoolean(PROP_CONTROLLER_IGNORE_TOO_LOW_TPS, false)) {
+			return true;
+		} else if (stopReason == StopReason.TOO_MANY_ERRORS &&
+			properties.getPropertyBoolean(PROP_CONTROLLER_IGNORE_TOO_LOW_TPS, false)) {
```

Comment:
It is bug, please fix to `PROP_CONTROLLER_IGNORE_TOO_LOW_TPS` -> `PROP_CONTROLLER_IGNORE_ TOO_MANY_ERRORS`

Reply:
fixed it !